README.md

```diff
@@ -311,6 +311,9 @@ pip install -r requirements.txt
 You can then build the documentation by running ``make <format>`` from the
 ``docs/`` folder. Run ``make`` to get a list of all available output formats.
 
+If you get a katex error run ```npm install katex```.  If it persists, try
```

Comment:
cc @jlin27 is it OK for adding this?

Reply:
Yep this addition is okay. 

I think you'll need to remove the dash in front of the second set of instructions though. 
```- npm install -g katex```